lib/style/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * styles
+ */
```

Comment:
Nice. I'm curious to see how we can play around with conventions around styling and styled components, and how to structure the files